db/re/item_combos.yml

```diff
@@ -25125,7 +25133,7 @@ Body:
           - Hawkeye     # 4879
     Script: |
       bonus bMaxSPrate,50;
-      bonus2 bAddRace,RC_All,-10;
+      bonus bAtkRate,-10;
```

Comment:
```suggestion
      bonus2 bAddClass,Class_All,10;
```

Reply:
I think this is correct (referring to jro, combining them will cancel all card effects)